src/AstToCFlat.ml

```diff
@@ -405,7 +405,7 @@ and mk_expr (env: env) (locals: locals) (e: expr): locals * CF.expr =
 
   | EBufCreate (l, e_init, e_len) ->
       if not (e_init.node = EAny) then
-        Warnings.fatal_error "init node is not any but %a (see SimplifyWasm)\n" pexpr e_init;
+        Warn.fatal_error "init node is not any but %a (see SimplifyWasm)\n" pexpr e_init;
```

Comment:
I don't know if it's feasible but is there a way to use -pack or something for compiler-libs to avoid all those pesky renamings? For instance this: https://opam.ocaml.org/packages/ocaml-compiler-libs/ seems to expose names that don't conflict.